core/src/test/resources/configuration/smart-testing.yml

```diff
@@ -9,3 +9,4 @@ scm:
     range:
       head: HEAD #<!--7-->
       tail: HEAD~2 #<!--8-->
+customStrategiesDefinition: ./custom.properties #<!--9-->
```

Comment:
Referencing the file when the content is very simple doesn't make sense to me. 
I would rather use: 
```yaml
customStrategiesDefinition:
    - smart.testing.strategy.cool=my.company.smart.testing:strategy-cool:1.0.0
```
The external property file seems to me just too complicated

Reply:
you mean the name of the property? Actually I am open to better name.

About what you mention, I think that a file is better because then you are able to share it between different projects, and not having to do copy-paste between projects. i see for example that here you'd be able to put a URL so a company can have stored on git and jsut put there the URL where these definitions are stored.

What we can do is allow both ways for now file and embedded.

Reply:
Nope, I didn't mean the property name. But yeah, if we had it only embedded, then I would use only `customStrategies`.
I would still vote for having it inside of the file and keep the config consistent. 

> I think that a file is better because then you are able to share it between different projects, and not having to do copy-paste between projects.

You could say the same thing about `pom.xml` dependencies or other stuff. I'm not sure if it is really useful. I would keep it simple and if there is a request for such a feature then yeah, we can implement it.

Reply:
so just embedded right?

Reply:
+1 for Embedded. 
>
> ```yaml 
> customStrategiesDefinition:
>   - smart.testing.strategy.cool=my.company.smart.testing:strategy-cool:1.0.0
> ```

@MatousJobanek Do we really need to define complete `smart.testing.strategy.cool`? 

How about 
``` yaml
customStrategies:
- cool=my.company.smart.testing:strategy-cool:1.0.0
```

Reply:
Well, the short version looks much better! The only thing I'm afraid of is that we are not consistent with the parameter names in `strategies.properties` file and here in config YAML file. 
@lordofthejars WDYT about it? would that be hard to support both cases?

Reply:
For this version I won't touch this yet. I would advocate for talk internally and do change properly in all places if we agree